README.md

```diff
@@ -1,12 +1,84 @@
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 
-<h1 align="center">
-  State of Dev in Morocco Source Code 
-</h1>
+[![All Contributors](https://img.shields.io/badge/all_contributors-11-orange.svg?style=flat-square)](#contributors-)
 
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+<p align="center">
+  <a href="https://www.geeksblablas.com">
+  <img width="477" alt="logo" src="https://user-images.githubusercontent.com/11137944/101990595-01f5f280-3ca8-11eb-8873-95a6234fb096.png">
+  </a>
+</p>
+<h2 align="center">
+  State Of Dev in Morocco Website
+</h2>
+<hr />
 
-## 🚀 How to Contribute
+StateOfDev.ma is a survey centered around software developers in Morocco by DevC Morocco Community.
 
-To be done :) 
+we wanted to know how we can help and support each other, and overall, be able to better respond to developers’ evolving needs.
 
-To fix typos, You can find all Questions in the `survey` folder.
+The website is built using [Gatsbyjs](http://gatsbyjs.org)
 
+## 🚀 Quick start
+
+1.  **Fork and clone the project**
+
+    ```sh
+    git clone git@github.com:your-username/stateofdev.ma.git
+    ```
+
+1.  **Start developing.**
+
+    Navigate into your new site’s directory and start it up.
+
+    ```sh
+    cd stateofdev.ma/
```

Comment:
Are we sure the . in the folder name will not create a issue in some OS ? In Mac it works and Linux i'm quite sure it's fine. What about Windows ?

Reply:
should work on windows too